impeller/renderer/backend/gles/render_pass_gles.cc

```diff
@@ -470,6 +474,45 @@ struct RenderPassData {
     }
   }
 
+  // When we have a resolve_attachment, MSAA is being used. We blit from the
+  // MSAA FBO to the resolve FBO, otherwise the resolve FBO ends up being
+  // incomplete (because it has no attachments).
+  //
+  // Note that this only works on OpenGLES 3.0+, or put another way, in older
+  // versions of OpenGLES, MSAA is not currently supported by Impeller. It's
+  // possible to work around this issue a few different ways (not yet done).
+  //
+  // TODO(matanlurey): See https://github.com/flutter/flutter/issues/137093.
+  if (!is_default_fbo && pass_data.resolve_attachment) {
```

Comment:
A missing cap check here for SupportsOffscreenMSAA for paranoia? Instead of the DCHECK for the blit.

Reply:
We already have cap checks in render target construction. We could add an FML CHECK here but I'd be worried that too many layers of error checking is going to make it hard to find/fix problems.